redis-memo.gemspec

```diff
@@ -26,4 +26,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rake'
   s.add_development_dependency 'rspec', '~> 3.2'
   s.add_development_dependency 'simplecov'
+  s.add_development_dependency 'rails'
```

Comment:
Actually, this would pull in too many dependencies we don't use. Let's use `ActiveRecord.version.to_s` instead.

Reply:
oh good catch! thanks!